### PR TITLE
Update configure-a-pallet.md

### PR DESCRIPTION
**Pull request by NukeManDan**
add the location of referenced code

- [ ] Are the audience and objective of the document clear? E.g. a document for developers that should teach them about transaction fees.
- [ ] Is the writing:
  - [ ] Clear: No jargon.
  - [ ] Precise: No ambiguous meanings.
  - [ ] Concise: Free of superfluous detail.
- [ ] Does it follow our style guide?
- [ ] If this is a new page, does the PR include the appropriate infrastructure, e.g. adding the page to a sidebar?
- [ ] Build the page ($ cd website && yarn start). Does it render properly? E.g. no funny lists or formatting.
- [ ] Do links go to rustdocs or devhub articles rather than code?
- [ ] If this PR addresses an issue in the queue, have you referenced it in the description?

### CODE REVIEW

**Review by wheresaddie — APPROVED** [review]

**Review by danforbes — CHANGES_REQUESTED** [review]
Can you please fix the line lengths to be consistent? I think we are using 100-character line lengths in this repo.

**Comment by wheresaddie** [comment]
you can run prettier and set it as an extension to run line length by default @NukeManDan if you are in vscode

File: docs/tutorials/add-a-pallet/configure-a-pallet.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ Every pallet has a component called `Trait` that is used for configuration. This
 interfaces in languages such as C++, Java and Go. FRAME developers must implement this trait for
 each pallet they would like to include in a runtime in order to configure that pallet with the
 parameters and types that it needs from the outer runtime. For instance, in the template pallet that
-is included in the Node Template, you will see the following `Trait` configuration trait:
+is included in the [Node Template](https://github.com/substrate-developer-hub/substrate-node-template), you will see the following `Trait` configuration trait:
+
```
Comment (JoshOrndorff) [inline docs/tutorials/add-a-pallet/configure-a-pallet.md:11]:
```suggestion
is included in the [Node Template](https://github.com/substrate-developer-hub/substrate-node-template),
you will see the following `Trait` configuration trait:

```

**Review by JoshOrndorff — APPROVED** [review]
Thanks!

**Review by danforbes — APPROVED** [review]
Thanks, guys :pray:

File: docs/tutorials/add-a-pallet/configure-a-pallet.md (hunk 1 above)
Comment (NukeManDan) [reply docs/tutorials/add-a-pallet/configure-a-pallet.md]: thanks for the line ending help 🙏🏼

**Comment by NukeManDan** [comment]
@JoshOrndorff you fixed the only 100+ line I think? We set now? 😁

**Comment by danforbes** [comment]
Is CI not running because this is coming from a fork? :thinking:

File: docs/tutorials/add-a-pallet/configure-a-pallet.md [hunk 2]
```diff
@@ -7,7 +7,10 @@ Every pallet has a component called `Trait` that is used for configuration. This
 interfaces in languages such as C++, Java and Go. FRAME developers must implement this trait for
 each pallet they would like to include in a runtime in order to configure that pallet with the
 parameters and types that it needs from the outer runtime. For instance, in the template pallet that
-is included in the Node Template, you will see the following `Trait` configuration trait:
+is included in the [Node Template](https://github.com/substrate-developer-hub/substrate-node-template),
+you will see the following `Trait` configuration trait:
+
+**`./pallets/template/src/lib.rs`**
```
Comment (danforbes) [inline docs/tutorials/add-a-pallet/configure-a-pallet.md:13]:
```suggestion
**`pallets/template/src/lib.rs`**
```

**Pull request merged**